docker-compose.dbtestdata.yaml

```diff
@@ -0,0 +1,31 @@
+version: "3.8"
+services:
+  db:
+    image: postgres:15
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: example
+    volumes:
+      - ./DB/:/docker-entrypoint-initdb.d/
```

Comment:
Since the services are called the same in both files they also share the implicit volume where the DB is persisted.
This means test data generation will only be successful if the volume was destroyed previously.
Was that intentional?

Reply:
fixed